README.md

```diff
@@ -13,24 +13,24 @@ In your application root directory, enter this command to install the connector:
 npm install loopback-connector-cassandra --save
 ```
 
-This installs the module from npm and adds it as a dependency to the application's `package.json` file.
+This installs the module from npm and adds it as a dependency to the application's `package.json` file.
```

Comment:
Noticing a bunch of seemingly unnecessary diffs here...are these tabs and not spaces?